.github/workflows/workflow-validation.yml

```
@@ -0,0 +1,29 @@
+name: Workflow Validation Testing
+
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
+      - 'feature/**'
+    paths:
+      - '.github/**.yml'
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+concurrency:
+  group: workflow-validation-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  visual-regression:
+    name: Run Workflow Validation
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
```

The check here ran in 17s so I think even 1 minute should be reasonable here :)
```suggestion
    timeout-minutes: 1
```